LetSwift/Helpers/Observable/Observable.swift

```diff
@@ -124,3 +124,14 @@ extension Observable where Element: Equatable {
         }
     }
 }
+
+extension Observable where Element == Void {
```

Comment:
In Swift 4 behavior of Void was changed. Right now, if method expects argument of type Void, it's needed to pass it explicitly (i.e. `(())`). As having `Observable<Void>` is quite common and `.next(())` looks weird, I decided to add those extensions

Reply:
This seems to be OK! Just like with the previous extension - remove the empty line 129.